exe_1.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+def findPermutations(str1, str2):
+	str1 = str1.lower()
+	str2 = str2.lower()
+
+	letters_and_frequencies_1 = make_letters_and_frequencies(str1)
+	letters_and_frequencies_2 = make_letters_and_frequencies(str2)
+	return compare_dicts(letters_and_frequencies_1, letters_and_frequencies_2)
+
+def make_letters_and_frequencies(string):
+	lf = {}
+	for char in string:
+		if char in lf:
+			lf[char] += 1
+		else:
+			lf[char] = 1
+	return lf
+
+def compare_dicts(dict1, dict2):
```

Comment:
Yes, that would be a problem, seeing that in the first case "a" would be in the dict2, but we would be missing "b" which is in dict2 and not in dict1. A solution to this would probably be to run another loop over dict2 same as that for dict1 before the "return True" line. Thanks for pointing this out, it is actually a huge bug in my code. Will try to be more careful 👍